apps/docs/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   // This file is not used in compilation. It is here just for a nice editor experience.
   "extends": "@tsconfig/docusaurus/tsconfig.json",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
I'm using WebStorm, and it's the IDE that suggested me to add this configuration to support TSX.